SpeechSDK/.gitignore

```diff
@@ -0,0 +1,3 @@
+/windows

```

Comment:
Hi, finally starting with review. First thing: why do you download SpeechSDK into this folder? It was intended only for README.md file. Everything was originally downloaded into target folder (determined by cargo injected variable **OUT_DIR**). I am not saying this is all wrong just want to understand reasons behind

Reply:
My thinking is: 
1. This eliminated the need for the library users to manually download the SDK themselves
2.  These folders are ignored by Git and won't get uploaded to the repo
3.  By extracting the speech SDK here, it can be reused and can survive the `cargo clean`, also users can set the `LD_LIBRARY_PATH`  to point to this dir( `./SpeechSDK/linux/sdk_output/lib/x64`) and run the example easily